plugins/org.jkiss.dbeaver.registry/src/org/jkiss/dbeaver/registry/DataSourceProviderDescriptor.java

```diff
@@ -110,7 +110,7 @@ public DataSourceProviderDescriptor(DataSourceProviderRegistry registry, IConfig
             this.treeDescriptor = this.loadTreeInfo(trees[0]);
         }
         this.supportsDriverMigration = CommonUtils.toBoolean(config.getAttribute("supports-migration"));
-        this.skipParentsAuthModels = CommonUtils.toBoolean(config.getAttribute("skipParentsAuthModels"));
+        this.inheritAuthModels = CommonUtils.toBoolean(config.getAttribute("inheritAuthModels"));
```

Comment:
I believe the new attribute name means the opposite thing - so it should be `true` by default.